src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilJudicialRoleMappingTest.java

```diff
@@ -198,68 +195,6 @@ void shouldReturnFeePaidRoles(String setOffice, String expectedRoles, String reg
         );
     }
 
-    @ParameterizedTest
-    @CsvSource({
-        "CIVIL Deputy Circuit Judge-Fee-Paid,fee-paid-judge"
-    })
-    void shouldReturnCircuitJudgeRoles(String setOffice, String roleNameOutput) throws IOException {
```

Comment:
the test scenario & all the unit code checks in `shouldReturnCircuitJudgeRoles` & `shouldReturnHmctsJudiciaryRoles` seem to be covered in [shouldReturnSalariedRoles](https://github.com/hmcts/am-org-role-mapping-service/blob/master/src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilJudicialRoleMappingTest.java#L70) & [shouldReturnFeePaidRoles](https://github.com/hmcts/am-org-role-mapping-service/blob/master/src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilJudicialRoleMappingTest.java#L138) hence why decided to remove

Reply:
Thansk for the rationlisation of hte test now that we are setting all relevant flags on all tests.